<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addStudent</class>
 <widget class="QMainWindow" name="addStudent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ögrenci ekleme sayfası</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>801</width>
      <height>601</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(188, 163, 127);</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>191</width>
       <height>311</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(234, 215, 187);</string>
        </property>
        <property name="text">
         <string> Öğrenci Numarası :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(234, 215, 187);
font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string> Ad :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(234, 215, 187);</string>
        </property>
        <property name="text">
         <string> Soyad :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(234, 215, 187);</string>
        </property>
        <property name="text">
         <string>Doğum tarihi :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="styleSheet">
         <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(234, 215, 187);</string>
        </property>
        <property name="text">
         <string>Cinsiyet :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="styleSheet">
         <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(234, 215, 187);</string>
        </property>
        <property name="text">
         <string>Sınıf  :</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QTextEdit" name="student_number_text">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>120</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </widget>
    <widget class="QTextEdit" name="student_name_text">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>170</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </widget>
    <widget class="QTextEdit" name="student_surname_text">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>220</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="dateTimeEdit">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>280</y>
       <width>194</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>390</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="add_btn">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>460</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 170, 0);
color: rgb(255, 255, 255);
font: 75 22pt &quot;MS Shell Dlg 2&quot;;
</string>
     </property>
     <property name="text">
      <string>Kaydet</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>320</y>
       <width>261</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QRadioButton" name="erkek">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>82</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 18pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Erkek</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="kadin">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>20</y>
        <width>82</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 18pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Kadin</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="mesaje">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>480</y>
       <width>261</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>14</pointsize>
       <weight>9</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>331</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(234, 215, 187);</string>
     </property>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>321</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string> Ögrenci ekleme sayfası</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
